fetch/mesh_data_test.go

```diff
@@ -707,3 +707,83 @@ func TestFetch_GetEpochATXIDs(t *testing.T) {
 		})
 	}
 }
+
+func toLayerIDs(nums []uint32) []types.LayerID {
+	result := make([]types.LayerID, 0, len(nums))
+	for _, num := range nums {
+		result = append(result, types.NewLayerID(num))
+	}
+	return result
+}
+
+func TestFetch_GetMeshHashes(t *testing.T) {
+	peer := p2p.Peer("p0")
+	errUnknown := errors.New("unknown")
+	tt := []struct {
+		name     string
+		params   [4]uint32 // from, to, delta, steps
+		expected []uint32
+		err      error
+	}{
+		{
+			name:     "success",
+			params:   [4]uint32{7, 23, 5, 4},
+			expected: []uint32{7, 12, 17, 22, 23},
```

Comment:
Would it make sense for `expected` to be of type `[]types.LayerID`, then we wouldn't need the function `toLayerIDs` to convert them in the test.